tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -206,6 +233,7 @@ private:
    std::string fContainerLabel{};
    std::shared_ptr<RDaosPool> fPool;
    ObjClassId_t fDefaultObjectClass{OC_SX};
+   uint32_t fCageSize = 16 * 64 * 1024;
```

Comment:
Maybe rename to `fDefaultCageSize` (to state that it might be overridden by user options)?
```suggestion
   uint32_t fDefaultCageSize = 16 * 64 * 1024;
```